airsonic-main/src/main/webapp/WEB-INF/jsp/playQueue.jsp

```diff
@@ -99,6 +99,40 @@
             }
         });
 
+        /** Toggle between <a> and <span> in order to disable play queue action buttons */
+        $.fn.toggleLink = function(newState) {
+            $(this).each(function(ix, elt) {
+
+                var node, currentState;
+                if (elt.tagName.toLowerCase() === "a") currentState = true;
```

Comment:
I think that you can use `elt.is("a")` instead.

Reply:
I tried, actually. Calling `$(elt)` results in a syntax error like `SyntaxError: '[object HTMLSpanElement]' is not a valid selector`.

I assume that's because we're using an old version of jQuery?

Reply:
We'll soon have the latest one via #930 :)